src/App.php

```diff
@@ -1200,4 +1177,79 @@ private function getRenderedPortals(): array
 
         return $portals;
     }
+
+    public function getResponse(): ResponseInterface
```

Comment:
The App cclass ode is quite long. Do you think it is a good idea to move all request/response code to `App\Request` and `App\Response`?

Reply:
I had the same idea, but we can opt for something like App\Http\Response|Request to avoid new classes on root namespace, what do you think?

Reply:
`App\Request` put into `App` directory seems exactly that solution ;-)

Reply:
Let's keep it as is for now...